required/tools/tools.ins

```diff
@@ -195,30 +195,22 @@ given in the file `manifest.txt'.
   \file{r.tex}{\from{fileerr.dtx}{run}}
   \file{x.tex}{\from{fileerr.dtx}{exit}}}
 
+\generate{\file{rename-to-empty-base.tex}{\from{fileerr.dtx}{return}}}
+
 
 \Msg{***********************************************************}
 \Msg{*}
 \Msg{* To finish the installation you have to move the following}
 \Msg{* files into a directory searched by TeX:}
 \Msg{*}
 \Msg{* All the files with extension `.sty' and `.tex'}
-\Msg{* Note there also may be a file .tex which is `invisible'}
-\Msg{* on some operating systems.}
+\Msg{* TeX may be blocked from writing .tex so writing rename-to-empty-base.tex}
+\Msg{* Rename that file to .tex after the docstrip generation.}
```

Comment:
Seems the quotation marks didn't make it into https://github.com/latex3/latex2e/commit/3e8ee83836f6a5db69ff9de4b765c8ef1a554705 :). Also, maybe a minor rewording to clarify that the file name `.tex` is really meant and not the file extension:

```suggestion

\Msg{* TeX may be blocked from writing a file called `.tex', so writing `rename-to-empty-base.tex'.}
\Msg{* Rename that file to `.tex' after the docstrip generation.}
```

Reply:
@user227621  not so keen on that as with the default log line length it wraps  after `rename-to-e`  I'll experiment a bit

Reply:
@user227621  better? 6b4a64c3a

Reply:
Looks good to me